scripts/automation/trex_control_plane/interactive/trex/common/services/trex_service_ap.py

```diff
@@ -362,10 +362,10 @@ def process_capwap_ctrl(self, rx_bytes, ap):
                 capwap_assemble['buf'] += control_str
 
                 if rx_pkt.is_last_fragment():
-                    capwap_assemble['assembled'] = CAPWAP_CTRL(
+                    capwap_assemble['assembled'] = bytes(CAPWAP_CTRL(
```

Comment:
Can you fix the same here https://github.com/cisco-system-traffic-generator/trex-core/blob/master/scripts/automation/trex_control_plane/interactive/trex/wireless/trex_wireless_worker.py#L1091 ?